MapboxGeocoder/MBRectangularRegion.swift

```diff
@@ -52,10 +52,20 @@ open class RectangularRegion: CLRegion, Codable {
         try container.encode(northEast, forKey: .northEast)
     }
     
+    #if swift(>=4.2)
+    open override var hash: Int {
+        var hasher = Hasher()
```

Comment:
I think you can drop this function all together for Swift 4.2 because it should be generated by the Swift compiler.
Can you verify this somehow?

Reply:
What you can do is, store the `RegionRegion.hash` output Int for a specific rectangular region.
Remove the following code:
```swift
open override var hash: Int {
        var hasher = Hasher()
        hasher.combine(southWest.latitude.hashValue)
        hasher.combine(southWest.longitude.hashValue)
        hasher.combine(northEast.latitude.hashValue)
        hasher.combine(northEast.longitude.hashValue)
        return hasher.finalize()
    }
```

Get the hash value for the same rectangular region and compare to previous result.

Reply:
Yeah, there are no build errors when I remove the hash declaration for Swift 4.2.